project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -154,8 +158,23 @@ private static PlacesSearchResult createTestPlacesSearchResult(String placeId) {
     return searchResult;
   }
 
+  private static ArgumentMatcher<FakeSearchRequest> matchExpectedSearchRequest(
```

Comment:
nit: matchesSearchRequest

* Expected is redundant here
* matches is usually combined in a sentence-like statement.

Reply:
makes sense